### ISSUE TITLE
Это classified as NOUN

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Russian-SynTagRus#2]
Not sure where the bug is here, either in the documentation (where it says that это is a demonstrative pronoun) or in the treebank where it is only as a noun...

```
$ cat ~/source/UniversalDependencies/UD_Russian-SynTagRus/*.conllu | grep -P '\tЭТО\t' | cut -f3,4 | sort -f | uniq -c | sort -gr 
   5742 ЭТО  NOUN
     54 ЭТО  PART

$ cat ~/source/UniversalDependencies/UD_Russian-SynTagRus/*.conllu | grep -P '\tЭТОТ\t' | cut -f3,4 | sort -f | uniq -c | sort -gr 
   4950 ЭТОТ    DET
    249 ЭТОТ    ADJ
      6 ЭТОТ    NOUN
```

Indeed out of the top 10 nouns, I would say that ЭТО, ТО and ВСЕ should probably be PRON, and РОССИЯ should be PROPN (although there are no proper nouns).

**Comment by dan-zeman** [comment]
It looks like a bug in the data. The original SynTagRus does not distinguish pronouns from nouns, and the `PRON` tag ought to be added during conversion.

**Comment by Kira-D** [comment]
fixed in 2.1
